### ISSUE TITLE
Enabling https in PHP in docker

### ISSUE THREAD

**Issue by zuphilip** [UB-Mannheim/ocr-fileformat#51]
It seems that currently URL with "https" are not working when using the Dockerfile, e.g.
```
[Sun Feb 12 16:29:32 2017] PHP Warning:  file_get_contents(): Unable to find the
 wrapper &quot;https&quot; - did you forget to enable it when you configured PHP
? in /ocr-fileformat/web/ocr-fileformat.php on line 85
[Sun Feb 12 16:29:32 2017] PHP Warning:  file_get_contents(https://cdn.rawgit.co
m/kba/ocr-fileformat-samples/master/samples/hocr/1.1/wetzel_reisebegleiter_1901_
0021.hocr): failed to open stream: No such file or directory in /ocr-fileformat/
web/ocr-fileformat.php on line 85
```
(Maybe also document it in installation notes.)

**Comment by zuphilip** [comment]
http://stackoverflow.com/questions/12946486/openssl-not-working-with-php-built-in-webserver

**Comment by kba** [comment]
Running the PHP builtin server is not optimal anyway, it was just convenient. Probably would be best to setup an nginx or apache to serve the PHP.

**Comment by zuphilip** [comment]
For testing/developing purposes the PHP builtin server mostly works, but it might be more stable to switch to another webserver like you suggested.

**Comment by zuphilip** [comment]
Fixed in #75.
